tests/integration/targets/snapshot_schedule/tasks/main.yml

```diff
@@ -5,21 +5,6 @@
     SC_PASSWORD: "{{ sc_password }}"
 
   block:
-    - name: Retrieve all snapshot schedules
-      scale_computing.hypercore.snapshot_schedule_info:
-      register: initial
-    - ansible.builtin.assert:
-        that:
-          - initial.records != []
```

Comment:
I don't think we need to have this test, it just opens up potential issues.
Check if any schedules exists, if not fail the test ... seems completely pointless to have this.
It's always going to fail on a new cluster and it doesn't check anything valuable.